### DISCUSSION TITLE
name update problem

### DISCUSSION THREAD

**Discussion by Oguzhanst** [drizzle-mizzle/CharacterAI-Discord-Bot#54] [Q&A]
When I use another discord, its name is not updated, it remains as the name of the bot.

**Comment by drizzle-mizzle** [comment]
I don't understand the question

**Reply by Oguzhanst** [reply]
When I upload to a new Discord channel, the bot's name and picture don't update.

**Reply by drizzle-mizzle** [reply]
> When I upload to a new Discord channel, the bot's name and picture don't update.

Have you allowed him to do it?

**Reply by drizzle-mizzle** [reply]
Picture should be updated no matter what, though. Think what you have done wrong. If you followed the guide step by step, it should just work.

**Reply by Oguzhanst** [reply]
It's working perfeclty but i already said that the other channels no its doesnt update espacially name tag

**Reply by drizzle-mizzle** [reply]
That's some problem with your channels or roles. Bot changes its nickname on the whole server and its picture in his actual bot profile.
